src/histolab/filters/image_filters_functional.py

```diff
@@ -380,7 +380,9 @@ def rag_threshold(
     n_segments: int = 800,
     compactness: float = 10.0,
     threshold: int = 9,
-) -> PIL.Image.Image:
+    mask: np.ndarray = None,
+    return_labels: bool = False,
+) -> Union[PIL.Image.Image, np.ndarray]:
```

Comment:
Union is not imported in the moule. You should import it ... also I would suggest to run test locally before pushing in order to be sure that all the tests are running locally 😺